main.go

```diff
@@ -149,7 +149,7 @@ func visit(url *url.URL) {
 	if err != nil {
 		log.Fatalf("unable to connect to host %vv %v", raddr, err)
 	}
-	fmt.Printf("\n%s%s\n", color.GreenString("Connected to "), color.CyanString(raddr.String()))
+	fmt.Fprintf(color.Output, "\n%s%s\n", color.GreenString("Connected to "), color.CyanString(raddr.String()))
```

Comment:
Can you please create a top level helper function called `printf` that handles this, then rewrite all the calls to fmt.Fprintf to use this.